aunt-leahs-app/src/components/TableComponent.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../styles.css';
+import { Table, TableBody, TableCell, Paper, TableHead, TableRow } from '@material-ui/core';
+
+// REQUIRES: data prop has to be array of objects where each object key represents the name of the field in camelcase.
+//					 Refer to data in constants.jsx
+//					 each field in data should be either string or boolean. boolean fields are displayed '✓' or '✗'
+
+const formatHeaderTitle = (title) => {
+	return title.replace(/([A-Z])/g, ' $1').replace(/^./, (str) => str.toUpperCase());
+};
+
+const TableComponent = (props) => {
+	const { data } = props;
+	let columnTitles = Object.keys(data[0]);
+	return (
+		<div className="tableComponent">
+			<Paper className="tableStyle">
+				<Table aria-label="simple table" className="table">
+					<TableHead className="tableHead">
+						<TableRow key="head">
+							{columnTitles.map((columnTitle) => (
+								<TableCell className="headerCell" align="left">
+									<b>{formatHeaderTitle(columnTitle)}</b>
+								</TableCell>
+							))}
+						</TableRow>
+					</TableHead>
+					<TableBody>
+						{data.map((row) => (
+							<TableRow key={row.firstName}>
+								{Object.keys(row).map((key) => (
+									<TableCell className="tableCell" align="left">
+										{typeof row[key] == 'boolean' ? row[key] ? '✓' : '✗' : row[key]}
```

Comment:
Try to use icons instead of special characters because those might not render properly based on device, OS, and browser

Reply:
I am able to use font awesome then, but the picture on Figma looks like special characters. 
I don't know if we should stick to the specification or use icon.

Reply:
Justin, you make a fair point, but I think that this is really a styling issue that we can return to later, if need be.